dotnet/CoreLib/AppBuilders/MemoryClientBuilder.cs

```diff
@@ -468,8 +480,11 @@ public Memory BuildServerlessClient()
                 this._memoryServiceCollection.AddTransient<TextPartitioningHandler>(serviceProvider
                     => ActivatorUtilities.CreateInstance<TextPartitioningHandler>(serviceProvider, "partition"));
 
-                this._memoryServiceCollection.AddTransient<SummarizationHandler>(serviceProvider
+                if (this._useSummarizeHandlers)
+                {
+                    this._memoryServiceCollection.AddTransient<SummarizationHandler>(serviceProvider
                     => ActivatorUtilities.CreateInstance<SummarizationHandler>(serviceProvider, "summarize"));
```

Comment:
nit: tab

Reply:
I try to be fastidious with whitespace.  Thanks for callout.  Fixed